docs/source/reference/pysupported.rst

```diff
@@ -934,7 +934,7 @@ The following built-in functions are supported:
   cannot be a function type (e.g. ``getattr(numpy, 'cos')`` is not supported as
   it returns a function type).
 * :func:`hash` (see :ref:`pysupported-hashing` below)
-* :class:`int`: only the one-argument form
+* :class:`int`
```

Comment:
Can you add the implementation limitations here? e.g. only support `str` argument.